spectra-generator/src/test/java/se/de/hu_berlin/informatik/gen/spectra/spectra/TraceCoberturaToSpectraTest.java

```diff
@@ -254,7 +254,7 @@ private void testOnProjectWithTestList(TestProject project, String outputDirName
                 .useJava7only(useJava7)
                 .setTimeout(timeout)
                 .setTestRepeatCount(testrepeatCount)
-                .setCustomJvmArgs(new String[] {MAX_HEAP, GC})
+                .setCustomJvmArgs(new String[]{MAX_HEAP, GC, "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5009"})
```

Comment:
why is this necessary?

Reply:
i used it for attaching a debugger and forgot to remove it